internal/repository/article/article.go

```diff
@@ -0,0 +1,85 @@
+package article
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"news/internal/entity"
+	"news/internal/repository/mgdb"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ArticleRep struct {
+	articleTitle *mongo.Collection
+	article      *mongo.Collection
```

Comment:
collection is like a table name in SQL = hardcoded, no (like a constant for instance)? or why would you need a different name? 🤔